doc/usage.rst

```diff
@@ -21,7 +21,9 @@ When executing partup, the following options can be specified:
 -h, --help              Show help options
 -c, --config=CONFIG     Layout configuration file in YAML format
 -d, --debug             Print debug messages
+-D, --debug-domains=DEBUG-DOMAINS     Comma separated list of modules to enable debug output for
 -p, --prefix=PREFIX     Path to prefix all file URIs with in the layout configuration
+-q, --quiet             Print only error messages, redirect messages to journal
 -s, --skip-checksums    Skip checksum verification for all input files
 -v, --version           Print the program version and exit
```

Comment:
Align the description of all commands please and wrap the text to 80 characters.